### PR TITLE
Styling fix and content update for Swaps

### PR DESCRIPTION
**Pull request by dan437**
## Explanation
- Fix unsupported "boxProps" prop in Swaps
- Use a simplified copy for clarity and readability

## Testing
- On the Build Quote page in Swaps in Advanced options the tooltip next to the text `Smart transaction` has spacing again
- On the View Quote page we show "M quotes" now as a link, e.g. "7 quotes"

### CODE REVIEW

File: ui/pages/swaps/fee-card/fee-card.js [hunk 1]
```diff
@@ -167,9 +166,7 @@ export default function FeeCard({
                   onClick={onQuotesClick}
                   className="fee-card__quote-link-text"
                 >
-                  {isBestQuote
```
Comment (mcmire) [inline ui/pages/swaps/fee-card/fee-card.js:170]: Any context behind this change we should know? Is this unreliable data or an unnecessary distinction?

**Review by mcmire — DISMISSED** [review]
Seems like a fine change, just a clarifying question.

File: ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap [hunk 2]
```diff
@@ -27,4 +27,4 @@ exports[`storiesMetadata matches expected values for storiesMetadata 1`] = `
...
   },
 }
-`;
```
Comment (adonesky1) [inline ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap:30]: 🤔

**Review by adonesky1 — APPROVED** [review]
LGTM

**Review by mcmire — APPROVED** [review]
Looks good!

**Comment by DONOVA28** [comment]
Thank you so much for your support call of the team

**Pull request merged**